### PR TITLE
Mention adding a reviewer in PR template.

### PR DESCRIPTION
**Pull request by pabloem**
r: @kennknowles 
I think we need to add this.

Also, perhaps we might want to make the whole PR template a bit more concise. What's your take?

### CODE REVIEW

**Comment by charlesccychen** [comment]
This may be problematic because including the text `R: @username` will implicitly make a mention on GitHub...

**Comment by pabloem** [comment]
Added backticks to avoid that issue.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -14,4 +14,5 @@ Follow this checklist to help us incorporate your contribution quickly and easil
  - [ ] Each commit in the pull request should have a meaningful subject line and body.
  - [ ] Run `./gradlew build` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
  - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).
+ - [ ] Once your change is ready to be reviewed, tag a reviewer (by adding `r: @username`) to start a code review process.
```
Comment (kennknowles) [inline .github/PULL_REQUEST_TEMPLATE.md]: nit: I like capital `R`

**Comment by kennknowles** [comment]
So go ahead and curate commits and merge when ready.

**Comment by pabloem** [comment]
I like it.

**Pull request merged**